app/src/main/java/com/github/multimatum_team/multimatum/model/Deadline.kt

```diff
@@ -30,14 +30,16 @@ enum class DeadlineState {
  * @property state the advancement state of the deadline
  * @property dateTime the time at which the work is due
  * @property description some description of the deadline, by default empty
+ * @property notificationsTimes ArrayList of the notification's trigger time before deadline
  * @constructor Creates a deadline from specified parameters
  * @throws IllegalArgumentException when title is empty or startDate > end
  */
 data class Deadline(
     val title: String,
     val state: DeadlineState,
     val dateTime: LocalDateTime,
-    val description: String = ""
+    val description: String = "",
+    val notificationsTimes: ArrayList<Long> = ArrayList()
```

Comment:
As before, unless I missed something type should be `List<Long>` (or maybe `MutableList` if you want to make sure that it is mutable)

Reply:
I think you're right